_overviews/scala3-book/scala-features.md

```diff
@@ -128,15 +128,13 @@ As Heather Miller states, Scala is considered to be a [strong, statically-typed
 
 - Correctness: you catch most errors at compile-time
 - Great IDE support
-    - Code completion
+    - Reliable code completion
     - Catching errors at compile-time means catching mistakes as you type
     - Easy and reliable refactoring
-    - Reliable code completion
 - You can refactor your code with confidence
 - Method type declarations tell readers what the method does, and help serve as documentation
-- Types make your code easier to maintain
-- Scalability: types help ensure correctness across arbitrarily large applications and development teams
-- Strong typing in combination with excellent inference enables mechanisms like [contextual abstraction]({{ site.scala3ref }}/contextual/motivation.html) that allows you to omit boilerplate code. Often, this boilerplate code can be inferred by the compiler, based on type definitions and a given context (e.g. method arguments for implicit parameters).
+- Scalability and maintainability: types help ensure correctness across arbitrarily large applications and development teams
```

Comment:
I have merged “types make your code easier to maintain” and “Scalability” into just “Scalability and maintainability”.